lib/mapping.gi

```diff
@@ -1597,3 +1597,14 @@ function(f,t)
     SetRange(t,Range(f));
   fi;
 end);
+
+InstallMethod( String, 
+    "for an IdentityMapping", 
+    [ IsEndoGeneralMapping and IsOne ], 
+    function( id ) 
+    local str; 
+    str := String( Source( id ) ); 
+    str := Concatenation( "IdentityMapping( ", str, " )" ); 
+    ConvertToStringRep( str );
```

Comment:
That shouldn't be needed, `Concatenation` for three inputs in string rep (which ought to be the case here) gives a string rep. Am I missing something?

Reply:
Removed ConvertToStringRep - it was just copied from some other methods.

Reply:
This isn't a requirement, just some GAP advice.

You could write `return StringFormatted( "IdentityMapping( {} )", Source( id ) );`. For this case, there isn't really much difference between Concatenation and StringFormatted, but StringFormatted can get more useful for longer strings.

Reply:
I don't understand the syntax, but have made the change.